File: EmbyStat.Web/ClientApp/src/app/movie/models/moviePersonStats.ts
import { Card } from '../../shared/models/card';
import { PersonPoster } from '../../shared/models/personPoster';

export class MoviePersonStats {
  public totalActorCount: Card;
  public totalDirectorCount: Card;
  public totalWriterCount: Card;
  public mostFeaturedActor: PersonPoster;
  public mostFeaturedDirector: PersonPoster;
  public mostFeaturedWriter: PersonPoster;
}

File: EmbyStat.Services/Models/Movie/MoviePersonStats.cs
using System;
using System.Collections.Generic;
using System.Text;
using EmbyStat.Services.Models.Stat;

namespace EmbyStat.Services.Models.Movie
{
    public class MoviePersonStats
    {
        public Card TotalActorCount { get; set; }
        public Card TotalDirectorCount { get; set; }
        public Card TotalWriterCount { get; set; }
        public PersonPoster MostFeaturedActor { get; set; }
        public PersonPoster MostFeaturedDirector { get; set; }
        public PersonPoster MostFeaturedWriter { get; set; }
    }
}
